app/models/cosmetic.rb

```diff
@@ -0,0 +1,7 @@
+class Cosmetic < ApplicationRecord
+  has_many :rates, dependent: :destroy
+  has_many :cart_items, dependent: :destroy
+  has_many :comments, dependent: :destroy
+  belongs_to :category, dependent: :destroy
```

Comment:
category sao lại bị xóa khi cosmetic xóa dc e. điều này là k nên. 1 category có nhiều cosmetic, sẽ ảnh hg đến thằng cosmetic khác
chỉ nên has_many dependent destroy thôi. còn belongs to thì k nên